templates/components/debug/debug_toolbar.html.twig

```diff
@@ -31,51 +31,20 @@
  # ---------------------------------------------------------------------
  #}
 
-{% set rand = rand|default(random()) %}
+<div id="debug-toolbar-applet"></div>
 
-{% macro widget_button(id, title, icon = null) %}
-   <li class="debug-toolbar-widget d-inline-block p-2" data-glpi-debug-widget-id="{{ id }}">
-      <button type="button" class="btn btn-icon border-0 p-1" title="{{ title }}" data-bs-toggle="tab">
-         {% if icon != null %}
-            <i class="{{ icon }} me-1"></i>
-         {% endif %}
-         <span class="debug-text"></span>
-      </button>
-   </li>
-{% endmacro %}
-
-<div id="debug-toolbar" class="position-fixed bottom-0 w-100 card" tabindex="0">
-   <div class="resize-handle mt-n2" style="cursor: row-resize; height: 10px; z-index: 1030"></div>
-   <div class="d-flex flex-row align-items-center">
-      <div class="debug-toolbar-badge d-flex">
-         <button type="button" class="btn btn-icon border-0 px-3 opacity-100 debug-logo" onclick="GLPI.Debug.showDebugToolbar();" disabled>
-            <i class="ti ti-bug"></i>
-         </button>
-      </div>
-      <div class="debug-toolbar-content d-flex w-100 justify-content-between align-items-center">
-         <ul class="debug-toolbar-widgets nav nav-tabs align-items-center border-0" data-bs-toggle="tabs">
-            {{ _self.widget_button('server_performance', 'Server performance', 'ti ti-clock-play') }}
-            {{ _self.widget_button('sql', 'SQL Requests', 'ti ti-database') }}
-            {{ _self.widget_button('requests', 'HTTP Requests', 'ti ti-refresh') }}
-            {{ _self.widget_button('client_performance', 'Client performance', 'ti ti-brand-javascript') }}
-         </ul>
-         <div class="debug-toolbar-controls">
-            <div class="debug-toolbar-control">
-               <button type="button" class="btn btn-icon border-0 p-1" name="toggle_content_area" onclick="GLPI.Debug.toggleExtraContentArea();">
-                  <i class="ti ti-square-arrow-up"></i>
-               </button>
-               <button type="button" class="btn btn-icon border-0 p-1" title="{{ __('Close') }}" onclick="GLPI.Debug.hideDebugToolbar();">
-                  <i class="ti ti-square-x"></i>
-               </button>
-            </div>
-         </div>
-      </div>
-   </div>
-   <div id="debug-toolbar-expanded-content" class="w-100 card d-none pe-2">
-   </div>
-</div>
-<script>
-    $(document).on('ready', () => {
-        window.GLPI.Debug.init({{ debug_info|json_encode|raw }});
+<script type="module">
+    const debug_toolbar_applet = window.Vue.createApp(window.Vue.components['DebugToolbar'].component, {
```

Comment:
Creating a Vue "app" just for the toolbar element. If you try creating an app and mounting it to the body element currently, the entire page will be blank. This seems to be caused by invalid HTML somewhere. One of those issues was solved recently where an `=` was missing between an attribute name and value.

There is no limitation keeping us from having multiple apps, so it is probably best to have one for each feature container element.

Reply:
As a side note, it would be very nice if we get the single app mounted to the body to work as it would allow us to use any external component anywhere without hassle.

For example, [tabler icons components](https://tabler.io/docs/icons/vue) or the [gitlab filter UX](https://gitlab-org.gitlab.io/gitlab-ui/?path=/story/base-filtered-search--default) (that we had to recreate from scratch for our kanban as we couldn't use their component without Vue).

Maybe not in this first pull request but something to keep in mind for later.